src/pp.js

```diff
@@ -131,16 +131,26 @@ function hasHardLine(doc) {
   });
 }
 
-function _makeIndent(n) {
+function _makeIndent(tabWidth) {
```

Comment:
We don't want to change this to tabWidth.

We want it specifically to represent a single space. I've been thinking about a few scenarios where we might indent something to an "odd" level: meaning, indent something 2 times, but add a space to make line up with something. Even people who use tabs sometimes have to mix spaces in there to make things line up appropriately.

Right now we have full control over indentation at the printer level. We would lose that with this, and for no gain. If we ever support tabs, it needs to be solely in the code that prints the indentation and it should convert 2 or 4 spaces to tab, and leave the rest. That's all you need to do.